atlp-rwanda/atlp-pulse-bn
# #424  in-app and email notifications for ratings

### What does this `PR` do ?
This `PR` fixes rating notifications in the in-app and email ways.
### Description of tasks that were expected to be completed
1. Coordinate rate a trainee,
2. Trainees get in-app notification
3. Trainees get email notification 
### Functionality
To test this fix you need:
1. Trainee account
2. coordinator account 
3. both in the same organization
### How has this been tested?
1. login as coordinator
2. rate one of your trainees
3. login as that rated trainee
4. you need to see in-app notification
5. check the trainee mail inbox for email notification
### PR Checklist:
- rating functionality
-In-app notification
- email-notification
### Track PR
Issue-#424
